testing/iframe.js

```diff
@@ -71,7 +71,7 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
     // on that window.
     window.beforeLoad = function(win) {
       // Flag as being a test window.
-      win.AMP_TEST = true;
+      win.AMP_TEST_IFRAME = true;
```

Comment:
I assume `AMP_TEST` belongs to the global window, not the testing iframe. I set it [here](https://github.com/ampproject/amphtml/pull/3699/files#diff-5a0269b2e709fa3dead48cccd5e86d5eR112)